.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules/
 temp/
+.idea
```

Comment:
Oops. Ignore this comment. I was reading the commits in reverse order and commented on an old commit. But my point is still valid :)